packages/components/src/components/amount-input/amount-input.scss

```diff
@@ -3,6 +3,7 @@
     flex-direction: column;
     padding: 0.8rem;
     height: auto;
+    width: 100%;
```

Comment:
Why we changing components package ? 🤔

Reply:
@heorhi-deriv you mean in legacy `Transfer`? 👀  Because in `wallets/package.json` there's no `components`

<img width="1177" alt="Screenshot 2023-12-06 at 10 45 54" src="https://github.com/binary-com/deriv-app/assets/119863957/0c82d928-2a15-47cf-8a8c-1f9f601bd35e">

Reply:
omg @thisyahlen-deriv 
you are right
sorry my fault
this width should be added only to ATMAmountInput
will do the changes 
thanks !!!🙏🏻